src/components/main/index.js

```diff
@@ -6,11 +6,18 @@ import { useLocation } from "react-router-dom";
 
 function getBackgroundImages(path) {
   const backgroundChangeList = {
-    "/create-wallet/status/updated": {
+    "/create-wallet/status/updated/consent-false": {
```

Comment:
I don't understand why you are doing this?

Reply:
@MDRKeil `/create-wallet/status/updated/:consentGiven`

Reply:
use this instead and then use the url param as the variable to show which card need to be shown, does that make sense?